segmentation/segmentation.cpp

```diff
@@ -594,11 +594,11 @@ void Segmentation::mergelistLoad(QTextStream & stream) {
             std::istringstream coordColorLineStream(stream.readLine().toStdString());
 
             uint64_t objId;
-            bool todo;
-            bool immutable;
+            bool todo = false;
+            bool immutable = false;
             Coordinate location;
             uint r; uint g; uint b;
-            uint64_t initialVolume;
+            uint64_t initialVolume = 0;
```

Comment:
these are simply not needed
and if it wants to init everything, why just 3 of them?

Reply:
It was suggested to initialize this variable, to silence the warning in 612, where it could be passed to createObjectFromSubobjectId uninitialized.

Reply:
but it can’t